tests/utils.py

```diff
@@ -169,7 +169,7 @@ def default_config(name, parse=False):
         "default_room_version": DEFAULT_ROOM_VERSION,
         # disable user directory updates, because they get done in the
         # background, which upsets the test runner.
-        "update_user_directory": False,
+        "update_user_directory_on": "",
```

Comment:
if the intention is to set this to a non-existent worker, might be better to make that clearer:

```suggestion
        "update_user_directory_on": "non-existent worker",
```